detox/test/src/Screens/DatePickerScreen.js

```diff
@@ -1,47 +1,49 @@
 import React, { Component } from 'react';
-import {
-  Text,
-  View,
-  StyleSheet,
-  DatePickerIOS
-} from 'react-native';
+import { Text, View, StyleSheet, DatePickerIOS } from 'react-native';
```

Comment:
You can't add `DatePickerIOS` on a shared component, it needs to run on both iOS and Android.
You might need to split this screen to .ios.js and .android.js.
This is probably the reason why the e2e hangs, the app crashes.

Reply:
But that’s not a new change. How did it work until now?

Reply:
ho, you're right, I haven't noticed that